src/Game.java

```diff
@@ -0,0 +1,114 @@
+package src;
+
+
+import java.util.Scanner;
+import java.util.Random;
+
+public class Game {
+    private Player player1;
+    private Player player2;
+
+    public Game(Player player1, Player player2) {
```

Comment:
Omdat je toch standaard null meegeeft, kun je de initialisatie in de constructor wel weglaten. De default waarde is toch null.